cross/android/arm64/tryrun.cmake

```diff
@@ -0,0 +1,128 @@
+# Need to verify this
+SET( HAVE_COMPATIBLE_EXP_EXITCODE 
```

Comment:
Link to CMakeCache.txt file: http://pastebin.com/pYChxXS1
From vanilla coreclr/master